src/App.js

```diff
@@ -1,220 +1,173 @@
 import React, { useState, useEffect, useRef } from 'react';
-import "react-datepicker/dist/react-datepicker.css";
+import 'react-datepicker/dist/react-datepicker.css';
 import './App.css';
 import './components/cell.css';
 import Cell from './components/cell';
-import Row from './components/row';
 
-const GRID_ROW_LENGTH =1;
-const GRID_COL_LENGTH =96;
+const GRID_ROW_LENGTH = 1;
```

Comment:
Have you noticed that adding multiple rows now doesn't work?

Part of the problem is when `<Row />` is called. It's missing a loop there.